product/views/Host.yaml

```diff
@@ -42,6 +42,9 @@ include_for_find:
   :compliances: {}
   :tags: {}
 
+# Skip using .references to avoid LEFT JOIN bomb (not needed)
```

Comment:
skip references to avoid left join with compliances - brings back too many extra records